02-peer-protocol.md

```diff
@@ -802,7 +825,12 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 A sending node:
   - MUST NOT offer `amount_msat` it cannot pay for in the
 remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+Fees") while maintaining its channel reserve
+  - if `option_anchor_outputs` applies to this commitment transaction and the sending
+    node is the funder:
+    - MUST be able to additionally pay for `to_local_anchor` and 
+      `to_remote_anchor` above its reserve. The anchor value should assumed to be
+      the maximum of the dust limits that have been negotiated by both parties.
```

Comment:
The PR currently proposes to honor each party's dust limit. If using always the initiator dust limit, it may be that the non-initiator has a tx in hands that contains an output below their dust limit.

Reply:
Okay right, is this what you intended with "also substract two times the local dust_limit_satoshis from the funder", shouldn't be "substract local `dust_limit_satoshis` + remote `dust_limit_satoshis` from the funder.

 Do you let a right to the funder to refuse opening a channel based on counter-party `dust_limit_satoshis` being to high to avoid inflation attack ?

Reply:
No it shouldn't be `local_dust_limit`+`remote_dust_limit`, because then the party that publishes the tx can still end up with an output that is below their dust limit.

Yes, the funder can cancel the channel creation if they don't like the dust limit that is in the `accept_channel` message.

Reply:
Gotcha, you're right, in BOLT3 context, local means the transaction builder and not the channel funder, that was the source of my confusion..